### DISCUSSION TITLE
Telegram

### DISCUSSION THREAD

**Discussion by namecallfilter** [ImranR98/Obtainium#1600] [Q&A] [answered]
I see that telegram is a source that you can add but when I give it the channel link it installs the actual telegram app, how do I add it as a source?

**Comment by ImranR98** [comment, marked as answer]
See #1
The source was named "Telegram App" to try and avoid this confusion 🫠

**Reply by namecallfilter** [reply]
wait so the app is the actual app and not scraping any apks ina  channel you give?
